src/main/js/src/containers/Quarantine/QuarantineHistory.js

```diff
@@ -132,39 +140,49 @@ const QuarantineHistory = props => {
               </div>
               <div className={classes.ListRow}>
                 {review.is_quarantined
-                  ? t('participationBan.banned')
+                  ? moment(review.updated).isAfter(review.exam_date, 'day')
```

Comment:
Pitäiskö ehto olla `!moment(review.updated).isBefore(review.exam_date, 'day')` jos ottaa huomioon tuon että peruutettu myöhässä jos peruutetaan myös tutkintopäivänä? Tai sit ottaa tästä myös negaation edestä pois ja kääntää noi true ja false keissit

Reply:
Vaikea sanoa mihin se raja pitäisi tarkalleen vetää.

Reply:
Backendin puolesta se raja taitaa nyt olla `cancel-registration-to-upcoming-exam!` queryssä siinä että osallistumisen peruutus tapahtuu, jos osallistumiskielto asetetaan ennen tutkintopäivää muttei enää tutkintopäivänä. Mietin että frontendissä ois hyvä jos UI reflektoisi tuota samaa logiikkaa.